less/components/illustration.less

```diff
@@ -1,15 +1,28 @@
 .illustration {
+  overflow: auto;
+  padding: 0 0 2rem 0;
   &.vertical-layout {
     flex-direction: column;
     .image-container {
       padding-right: 0;
     }
   }
-  .img-circle {
```

Comment:
`.img-circle` is a class defined in [Bootstrap](https://github.com/twbs/bootstrap/blob/3487898a6af8c1d5b12f7b040d30e71b19e34923/less/scaffolding.less#L106).  Since we have changed how classes get consume in the `<Illustration>` component I had to rename `.img-circle` into something else (ie. `.circular-image`) to avoid tons of styling override.